servers/tests/framework.rs

```diff
@@ -127,6 +125,7 @@ impl Default for LocalServerContainerConfig {
 /// A top-level container to hold everything that might be running
 /// on a server, i.e. server, wallet in send or receive mode
 
+#[allow(dead_code)]
```

Comment:
allow(dead_code) could theoretically be added up top with ! to avoid repetition, but that kind of defeats it purpose completely. The allow markings are actually not dead. Possibly there's another way by marking `pub` also on the module, but I didn't figure that out. If anyone knows or has an idea, say now or stay silent until some later date.

I'll be happy to clean this up further also later on. My first goal now is to have Travis build and test log outputs get a cleaner look.